helpers/utilities.go

```diff
@@ -817,10 +823,13 @@ func assertScanResults(
 ) ([]AssertionMismatch, error) {
 	var assertionFile string
 	if tc.Profile != "" {
-		assertionFile = fmt.Sprintf("%s-%s.yaml", tc.Profile, tc.Version)
+		assertionFile = path.Join(tc.ContentDir, assertionsPath,
```

Comment:
This needs to get fixed because it's not discovering the assertion file path already present in the content repository:

```
2025/09/26 04:18:51 Saved YAML data to /logs/artifacts/final-rhcos4-moderate-results.yaml
2025/09/26 04:18:51 No assertion file found: /tmp/content-1770658672/tests/assertions/ocp4/moderate-4.15.yaml
2025/09/26 04:18:51 Generated assertion file: /logs/artifacts/rhcos4-moderate-4.15-rule-assertions.yaml
2025/09/26 04:18:51 Generated assertions for 482 rules
```